collect_app/src/main/res/values/attrs.xml

```diff
@@ -18,4 +18,8 @@
     <declare-styleable name="Material3PolyFill">
         <attr name="colorSurfaceContainerLow" format="color" />
     </declare-styleable>
+
```

Comment:
Let's add a comment here so that it's clear these attributes are "extensions" to Material 3.

Reply:
Done.